cdap-ui/app/directives/dag-plus/my-dag.less

```diff
@@ -387,19 +422,23 @@ my-dag-plus {
         left: 10vw;
       }
 
-      &.errortransform {
-        color: @error-transform;
-        left: 30vw;
-      }
-
       &.realtimesink,
       &.batchsink,
-      &.alertpublisher,
-      &.sparksink {
+      &.sparksink,
+      &.errortransform,
+      &.alertpublisher {
```

Comment:
if you just need the `left` property for errortransform and alertpublisher, it's much better to just redefine that than overwriting the color later on. The CSS generated will be cleaner